spec/factories/user.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :user do
+    email "paullorijgu@gmail.com"
+    full_name  "Laul Guberson"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.